indicators/models.py

```diff
@@ -1103,49 +1104,66 @@ class Indicator(SafeDeleteModel):
     # this includes a relationship to a country
     strategic_objectives = models.ManyToManyField(
         StrategicObjective, verbose_name=_("Country Strategic Objective"),
-        blank=True, related_name="strat_indicator", help_text=" "
+        blank=True, related_name="strat_indicator",
+        help_text=_("Identifying the country strategic objectives to which an indicator contributes, allows us to "
+                    "filter and analyze related sets of indicators. Country strategic objectives are managed by the "
+                    "TolaData country administrator.")
     )
 
-    name = models.CharField(verbose_name=_("Name"), max_length=500,
-                            null=False, help_text=" ")
+    name = models.CharField(verbose_name=_("Name"), max_length=500, null=False,
+                            help_text=_("Provide an indicator statement of the precise information needed "
+                                        "to assess whether intended changes have occurred."))
 
     number = models.CharField(
         _("Number"), max_length=255, null=True, blank=True, help_text=" "
     )
 
     source = models.CharField(
-        _("Source"), max_length=255, null=True, blank=True, help_text=" "
+        _("Source"), max_length=255, null=True, blank=True,
```

Comment:
This would be a good one to have a translator comment on since it's just the one word.